src/lib/features/frontend-api/frontend-api-controller.ts

```diff
@@ -248,28 +240,12 @@ export default class FrontendAPIController extends Controller {
             return;
         }
 
-        const environment =
```

Comment:
in the spike controller was doing orchestration of services but in the prod ready version we want controller to use only one service (frontend API service) doing the orchestration